cdk/src/monitoring/synthetics.handler.ts

```diff
@@ -0,0 +1,50 @@
+const synthetics = require('Synthetics');
+const log = require('SyntheticsLogger');
+const syntheticsConfiguration = synthetics.getConfiguration();
+
+exports.handler = async function () {
+  // TODO: cycle through several addresses
+  const addresses = ['5607 HOME LN, TOLEDO OH 43623'];
+  const address = addresses[0];
+
+  const url = `https://${process.env.DOMAIN}/`;
+
+  syntheticsConfiguration.setConfig({
+    includeRequestHeaders: true, // Enable if headers should be displayed in HAR
+    includeResponseHeaders: true, // Enable if headers should be displayed in HAR
+    restrictedHeaders: [], // Value of these headers will be redacted from logs and reports
+    restrictedUrlParameters: [], // Values of these url parameters will be redacted from logs and reports
+  });
+  const page = await synthetics.getPage();
+
+  // TODO: map these steps to CUJs.
+
+  // Navigate to the initial url
+  await synthetics.executeStep('navigateToUrl', async function (timeoutInMillis = 30000) {
+    await page.goto(url, { waitUntil: ['load', 'networkidle0'], timeout: timeoutInMillis });
+  });
+
+  // Fill and search for an address.
+  await synthetics.executeStep('searchAddress', async function () {
+    await page.waitForSelector("[id='app']", { timeout: 30000 });
+    await page.screenshot();
+    const searchInput = await page.waitForSelector('input.mapboxgl-ctrl-geocoder--input');
+    await searchInput.type(address);
+    const suggestion = await page.waitForSelector('.mapboxgl-ctrl-geocoder--suggestion-title');
+    await page.screenshot();
+    await suggestion.click();
+    // This selects the first button with this class.
+    // TODO: add and use element IDs.
+    const searchButton = await page.waitForSelector('#scorecard-search-button');
```

Comment:
Curious if this is the recommended way of identifying buttons (via ids) rather than their text? This would mean we have to add ids for the sole purpose of testing

Reply:
Good point, and I don't know if it's the best way of identifying elements. I did a quick Google and Moma search for best practices and found [this talk](https://docs.google.com/presentation/d/19sqLP_cuBPWK7ol5SrUaQL7_L0hpn1u0ewmODXFg42Q/edit#slide=id.p), which didn't seem to state a preference (it identified elements by class or id in the examples).

I'll leave it as-is and let devs decide how they want to in future tests.